### ISSUE TITLE
Firefox : md-tooltip don't display on a md-list-item

### ISSUE THREAD

**Issue by mpicciolli** [angular/material#8172]
Hello,

On Firefox, md-icon tooltip don't display when it's placed on a md-list-item element

Codepen : [http://codepen.io/mpicciolli/pen/QNBMqg](http://codepen.io/mpicciolli/pen/QNBMqg)

angularjs/1.4.8
angular-material/v1.0.5

Any idea why this doesn't work?

**Comment by scriptsure** [comment]
Ditto on chrome +1

**Comment by DevVersion** [comment]
Tracking with #8946.

_Fer22f commented:
> However, 1.1.0 fixes the href/ng-click disparity, and breaks md-tooltip. Seeing this is basically default behavior, it might make this issue a "feature request" instead. I found #8172, but it lacks information and is now stale.

**Comment by ashishpatel02** [comment]
Will it not work inside md-list-item?

**Comment by cmacdonnacha** [comment]
+1

**Comment by ashishpatel02** [comment]
It Won't But i have found one workaround which can help others.

If you inspect it then you will be surprised with `button` inside `md-list` so what i have done, i have added `button` manually inside `md-list-item` without using `md-list` tag and added `md-tooltip` underneath of that.

check below :

```
<md-list-item class="usercontentpad" ng-repeat="repeat with data">
    <button class="md-no-style md-button md-default-theme" ng-transclude="">
        <div class="md-list-item-inner ng-binding ng-scope">
            {{ item to display }}
        </div><div class="md-ripple-container" style=""></div>
    </button>
    <md-tooltip md-direction="right" class="large" style="position: absolute;">
            {{tooltip to display}}
    </md-tooltip>
</md-list-item>
```

**Comment by cmacdonnacha** [comment]
Thanks for that @ashishpatel02 . Is there anyway way to have it on the list item itself instead of the button?

**Comment by ashishpatel02** [comment]
you can replace button with anything but i'm not sure it will work or not, you can try this with codepen first for the small demo @cmacdonnacha

**Comment by mpicciolli** [comment]
@ashishpatel02 Your solution doesn't work. Do you have a codepen ?

**Comment by ashishpatel02** [comment]
@mpicciolli here it is => http://codepen.io/ashuslove/pen/bZypRE?editors=101#anon-login

**Comment by skortchmark9** [comment]
agreed this would be nice to have.

**Comment by Splaktar** [comment]
This seems related to PR https://github.com/angular/material/pull/11539. Please let me know if this is still an issue after that is merged (targeting `1.1.12`) by opening a new issue with a CodePen reproduction.
